roles/tendrl-ansible.tendrl-server/tasks/tendrl-monitoring-integration.yml

```diff
@@ -80,6 +80,17 @@
   notify:
     - restart grafana-server
 
+- name: Configure grafana root_url with sub_path for reverse proxy
+  lineinfile:
```

Comment:
if the root_url is commented then is this task will uncomment it?

Reply:
it will change line but it won't un-comment right?

Reply:
;root_url = http://localhost:3000  this line should come like
root_url = http://localhost:3000/grafana

; should removed

Reply:
@GowthamShanmugam , this task is specifically required only in case of upgrade. In my understanding, in case of fresh install, its already taken care. So the question here is in upgrade scenario is there a chance where roo_url is commented?
@dahorak suggestion looks better otherwise

Reply:
it works fine, this task is taking care of un-comment also, sorry for the confusion :) i will modify the PR based on @dahorak comment

Reply:
@GowthamShanmugam so you mean this will work fine both in fresh setup as well as upgrade, right?

Reply:
@shtripat Yes, it should ensure, that the option `root_url` with proper value is in the right section of the configuration. If it is already present (new installation), it will do no change. If it is not there (update scenario) it will add it.

Reply:
@shtripat yes it will work in both the case, i tested

Reply:
Ack